### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by monkstone**
About time to bump versions following recent polyglot-maven release, and more recent jruby version.

### CODE REVIEW

**Comment by monkstone** [comment]
There doesn't seem to be any activity to cover java integration I thought I might give it a stab?

File: .mvn [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kares) [inline .mvn:1]: I am never sure which is generated and fine to include and which is not. 
this seems like polyglot mvn's generated temporary piece (due pom.rb)?

Comment (monkstone) [reply .mvn:1]: From my understanding the `.mvn` folder and extensions.xml is required for maven to recognize that we are using polyglot maven (pom.rb) and not regular maven (pom.xml), the change I proposed just bumps the version, for completeness we generate pom.xml but it is not actually used.
https://github.com/takari/polyglot-maven-examples/blob/master/ruby/.mvn/extensions.xml

**Comment by monkstone** [comment]
PS: if @kares are happy with latest @enebo merge, I could withdraw add on PR re java-integration which is mainly for discussion/wip

File: .mvn (hunk 1 above)
Comment (monkstone) [reply .mvn:1]: @kares It seem things are not as straightforward as I thought, as the latest version of maven does not seem to require the `.mvn` folder, and `pom.rb` is sufficient. But for the moment we are probably safer following takari master.

**Comment by monkstone** [comment]
Closing as discussed

**Pull request closed without merging**